### DISCUSSION TITLE
Not Quite Fully Functional Circuitry

### DISCUSSION THREAD

**Discussion by QuietImCoding** [TodePond/CellPond#349] [Show and tell]
Been working on some simple electronic logic gates for cellpond, would be fun to make a little tiny computer in a pond! Very buggy at the moment but gray is wire, yellow is electricity, then there is an on/off lightbulb, and an AND, OR, and NOT logic gate in that order. 

Have fun trying it out! Let me know if anyone has any ideas on how to improve things.

[circuitry.json](https://github.com/TodePond/CellPond/files/15439926/circuitry.json)

**Comment by CkreaM** [comment]
Woah, can you tell how to use each colour??
